nbdev/quarto.py

```diff
@@ -147,8 +147,10 @@ def nbdev_readme(
     _render_readme(cfg_path, path, chk_time)
     if (tmp_doc_path/'README.md').exists():
         _rdm = cfg_path/'README.md'
+        _rdmi = tmp_doc_path/(Path(get_config().readme_nb).stem + '_files')
         if _rdm.exists(): _rdm.unlink() # py37 doesn't have arg missing_ok so have to check first
         move(str(tmp_doc_path/'README.md'), cfg_path) # README.md is temporarily in nbs/_docs
+        if _rdmi.exists(): move(str(_rdmi), cfg_path) # Move Supporting files for README
```

Comment:
If move() can't take a str we should modify jt

Reply:
sure

Reply:
@jph00 this isn't our move, this is `move` from shutil, [which doesn't accept Path like objects before Python 3.9](https://docs.python.org/3/library/shutil.html#shutil.move)

BTW - I saw this in the code elsewhere other than this diff, and assumed that this is was for a reason so I followed the convention - looks like it was justified

Reply:
Unless I'm missing something, it seems to be ours?

```
from fastcore.shutil import rmtree,move
```

Reply:
After looking again, I do think the `str` should be able to be removed -- try it and see if it works.

Reply:
Yeah it works 🎉